.github/workflows/production-suite.yml

```diff
@@ -125,6 +125,11 @@ jobs:
             is_buster: true
             os: buster
 
+          - docker_image: ligmitz/ci:bullseye
```

Comment:
Once this is working, we should remember to move this container to the `zulip` organization before merging.

Reply:
Yes, will keep in mind.

Reply:
I uploaded a  `zulip/ci:bullseye` base image.